frontend/test/metabase/scenarios/collections/bookmarks.cy.spec.js

```diff
@@ -0,0 +1,92 @@
+import _ from "underscore";
```

Comment:
It's strange that linter doesn't complain, but I don't see `underscore` being used anywhere in this spec?

Reply:
Yes, strange miss by the linter. 

Removed.

Reply:
Yes, very strange. Removed.